opdracht-1/styles.css

```diff
@@ -9,33 +9,44 @@ body {
 }
 
 /* 1.1 Geef de 'img' tag in the header een ID en zet zijn breedte op 100% */
-
-
+#image-header{
```

Comment:
12: Je spreekt hier image-header aan met een id, terwijl je het een class hebt gegeven in html. Hierdoor werkt deze code niet en is de image size niet aangepast